### ISSUE TITLE
ID Lookup button finds wrong codes

### ISSUE THREAD

**Issue by bwanaaa** [charlestytler/streamdeck-dcs-interface#62]
```
NMSP button      button ID used in example code  for A10C             button ID used in lookup (which fail)
STR PT                         612                                                                611
ANCHR                          614                                                                613          
TCN                            616                                                                615
ILS                            618                                                                617

```

Interestingly, the code for the A10C works properly for the A10C_2. But if you make buttons using the A10C_2 lookups, they fail. Also interesting, looking at all the button IDs in "DCS Comms" shows all the above button IDs. But searching for "NMSP" with "ID Lookup" fails to show the buttons used in the example code. Perhaps Eagle Dynaminics forgot to label these 'properly working' buttons so the "ID Lookup" does not show them. At all.
You cannot even find them using the dcs-clickable-data-extract tool

I wonder how many other functions in the A10C example code use 'undocumented' button IDs?
Creating a blank momentary with text and inputting random numbers into the "Title Text Change on DCS Update Settings" "DCS ID" field gives interesting results. And you can change the format of the displayed text by using the other button types that use text.

**Comment by bwanaaa** [comment]
Solved by comment posted here:
https://github.com/charlestytler/streamdeck-dcs-interface/issues/61
It turns out that ID lookup only searches for pushable buttons. The lamps have separate button IDs. They are documented.

**Comment by bwanaaa** [comment]
yes thank you. That solved issue #62 as well.

**Comment by charlestytler** [comment]
@bwanaaa thanks for following up and closing out the issues!
